examples/07_advanced/plot_beta_series.py

```diff
@@ -0,0 +1,354 @@
+# Beta series can be used much like resting-state data, though generally with
+# vastly reduced degrees of freedom than a typical resting-state run, given
+# that the number of trials should always be less than the number of volumes
+# in a functional MRI run.
+#
+# Two common applications of beta series are to functional connectivity and
+# decoding analyses.
+# For an example of a beta series applied to decoding, see
+# :ref:`sphx_glr_auto_examples_02_decoding_plot_haxby_glm_decoding.py`.
+# Here, we show how the beta series can be applied to functional connectivity
+# analysis.
+# The following example is based on
+# :ref:`sphx_glr_auto_examples_03_connectivity_plot_seed_to_voxel_correlation.py`,
+# which goes into more detail about seed-to-voxel functional connectivity
+# analyses.
+import numpy as np
+
+from nilearn.maskers import NiftiSpheresMasker, NiftiMasker
+
+# Coordinate taken from Neurosynth's 'language' meta-analysis
+coords = [(-54, -42, 3)]
+
+# Initialize maskers for the seed and the rest of the brain
+seed_masker = NiftiSpheresMasker(
+    coords,
+    radius=8,
+    detrend=True,
+    standardize=True,
+    low_pass=None,
+    high_pass=None,
+    t_r=None,
+    memory='nilearn_cache',
+    memory_level=1,
+    verbose=0,
+)
+
+brain_masker = NiftiMasker(
+    smoothing_fwhm=6,
+    detrend=True,
+    standardize=True,
+    low_pass=None,
+    high_pass=None,
+    t_r=None,
+    memory='nilearn_cache',
+    memory_level=1,
+    verbose=0,
+)
+
+# Perform the seed-to-voxel correlation for the LSS 'language' beta series
+lang_seed_beta_series = seed_masker.fit_transform(lss_beta_maps['language'])
+lang_beta_series = brain_masker.fit_transform(lss_beta_maps['language'])
+lang_corrs = np.dot(
+    lang_beta_series.T,
+    lang_seed_beta_series,
+) / lang_seed_beta_series.shape[0]
+language_connectivity_img = brain_masker.inverse_transform(lang_corrs.T)
+
+# Perform the seed-to-voxel correlation for the LSS 'string' beta series
+string_seed_beta_series = seed_masker.fit_transform(lss_beta_maps['string'])
+string_beta_series = brain_masker.fit_transform(lss_beta_maps['string'])
+string_corrs = np.dot(
+    string_beta_series.T,
+    string_seed_beta_series,
+) / string_seed_beta_series.shape[0]
+string_connectivity_img = brain_masker.inverse_transform(string_corrs.T)
+
+# Show both correlation maps
+fig, axes = plt.subplots(figsize=(10, 8), nrows=2)
+
+display = plotting.plot_stat_map(
+    language_connectivity_img,
+    threshold=0.5,
+    vmax=1,
+    cut_coords=coords[0],
+    title='Language',
+    figure=fig,
+    axes=axes[0],
+)
+display.add_markers(
+    marker_coords=coords,
+    marker_color='g',
+    marker_size=300,
+)
+
+display = plotting.plot_stat_map(
+    string_connectivity_img,
+    threshold=0.5,
+    vmax=1,
+    cut_coords=coords[0],
+    title='String',
+    figure=fig,
+    axes=axes[1],
+)
+display.add_markers(
+    marker_coords=coords,
+    marker_color='g',
+    marker_size=300,
+)
+
+fig.show()
```

Comment:
I thought that your point would be to compare LSA vs LSS ?

Reply:
I had considered it, but I was worried that anything I could show with one run of one specific task would be misleading, given that the tradeoffs between LSA and LSS should only become apparent when you compare across designs (e.g., fast vs. slow event-related tasks).

Reply:
Makes sense, but I'm just worried to leave users with yet another alternative without any information on how to choose one versus the other.

Reply:
I added a topic section on choosing the right model toward the top:

![image](https://user-images.githubusercontent.com/8228902/151836788-adc72407-fd1e-48b7-ada4-9dfe44c9cf26.png)

There are citations in there too, but they don't show up in the CI build.

Reply:
OK.